### DISCUSSION TITLE
We need your brain - AR communication

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [sceneview/sceneform-android#32] [General]
Hi everyone,

I open here a quick discussion to have anyone point of view, idea or answer on some points.

Here is my first question:

I really think that one of the most problem with AR is that people doesn't really know what it is.
And I'm thinking that a good way to help solving it could first be to change how AR is visually shown to none using it public.
In case of VR, people using virtual headsets are usually shown to make understand that the 3D rendering is what the user is viewing. 
In case of AR, in most of the communications we only see screenrecords or screenshots which doesn't actually make people understand that it is captured from a real environment that the user is in.
One first thing important, I think, is to capture from a most common real environment.

But the screen capture only in not enough.

Quick example : I shared the [Mission to Mars demo video](https://youtu.be/fT4-oZXsDDs) on my Facebook wall and asked my friend what they think the app is doing.
Most of them thought it was something like viewing 3D videos on the phone and that's all.
While it's an incredible app using most of the last AR capabilities.
In fact, I think the watcher brain actually view it like any 3D rendering in movies, commercials,...

So, what, you think, could be a good way to demonstrate AR on a screenrecord/screenshot ?

_Example Answer:_
_Taking a picture from another phone showing the actual one playing AR app and environment at the same time._

**Comment by issacclee** [comment]
In my experience, a screen capture video that showcase the entire process of selecting apps in the home menu, tap on apps, entering apps, initiate ar session, anchor environment and so on, would have more accuracy in terms of convey the idea of augmented reality to customer.

**Reply by ThomasGorisse** [reply]
@issacclee Thanks for your ideas
